packages/dropdown-react/src/Dropdown.tsx

```diff
@@ -49,13 +51,22 @@ export const Dropdown = ({
     errorLabel,
     inline = false,
     defaultPrompt = "Velg",
-    initialShow = false,
-}: Props) => {
+    variant,
+}: Props) {
     const [selectedValue, setSelectedValue] = useState(initialInputValue);
-    const [dropdownIsShown, setShown] = useState(initialShow);
+    const [displayedValue, setDisplayedValue] = useState(initialInputValue);
+    const [dropdownIsShown, setShown] = useState(false);
     const [listId] = useState(`dropdown${nanoid(16)}`);
     const hasSelectedValue = typeof selectedValue !== "undefined";
     const listRef = useListNavigation();
+    const componentClassName = "jkl-dropdown".concat(
+        inline ? ` jkl-dropdown--inline` : "",
+        dropdownIsShown ? ` jkl-dropdown--open` : "",
+        !hasSelectedValue ? ` jkl-dropdown--no-value` : "",
+        !!errorLabel ? ` jkl-dropdown--invalid` : "",
+        className ? ` ${className}` : "",
+    );
+    const labelClassName = "jkl-label".concat(variant ? ` jkl-label--${variant}` : "");
```

Comment:
I'm torn. I really like the `concat`-style for `componentClassName`, but not for `labelClassName`.

Reply:
Haha! I agree that it works better with multiple variables, though. How about just
```tsx
const labelVariantClass = variant ? ` jkl-label--${variant}` : "";
…
<label className={"jkl-label" + variantClass}>¡Hola!</label>
```
for the labels?